content/docs/next-release/operator.md

```diff
@@ -549,7 +549,7 @@ Note that the job loads all data for the current day into memory.
 
 ## Auto-injecting Jaeger Agent Sidecars
 
-The operator can inject Jaeger Agent sidecars in `Deployment` workloads, provided that the deployment or a namespace has the annotation `sidecar.jaegertracing.io/inject` with a suitable value. The values can be either `"true"` (as string), or the Jaeger instance name, as returned by `kubectl get jaegers`. When `"true"` is used, there should be exactly *one* Jaeger instance for the same namespace as the deployment, otherwise, the operator can't figure out automatically which Jaeger instance to use. A specific Jaeger instance name on a deployment has a higher precedence than `true` applied on a namespace.
+The operator can inject Jaeger Agent sidecars in `Deployment` workloads, provided that the deployment or it's namespace has the annotation `sidecar.jaegertracing.io/inject` with a suitable value. The values can be either `"true"` (as string), or the Jaeger instance name, as returned by `kubectl get jaegers`. When `"true"` is used, there should be exactly *one* Jaeger instance for the same namespace as the deployment, otherwise, the operator can't figure out automatically which Jaeger instance to use. A specific Jaeger instance name on a deployment has a higher precedence than `true` applied on it's namespace.
```

Comment:
Apologies - the two "it's" that I suggested to introduce should be "its" :)

Reply:
that makes more sense, I shouldn't be blindly copying